web/client/localConfig.json

```diff
@@ -656,8 +656,14 @@
         }, "Login", "Language", "NavMenu", "DashboardSave", "DashboardSaveAs", "Attribution", "Home", {
           "name": "Share",
           "cfg": {
-            "embedPanel": false,
-            "advancedSettings": false
+            "showAPI": false,
+            "advancedSettings": false,
+            "shareUrlRegex": "(h[^#]*)#\\/dashboard\\/([A-Za-z0-9]*)",
+            "shareUrlReplaceString": "$1dashboard-embedded.html#/$2",
+            "embedOptions": {
+              "showTOCToggle": false,
+              "showConnectionsParamToggle": true
```

Comment:
@tdipisa should this be true by default?

Reply:
In my opinion is better to keep this value false by default because is needed only for dashboard. If we put this value true by default we should instead add a rule to share plugin of map with showConnectionsParamToggle set to false.